### ISSUE TITLE
Some of the code for the operation in the article was not found. Where is the code for this part of the operation?

### ISSUE THREAD

**Issue by oula-le** [GhTara/Convolutional-Fuzzy-Neural-Network#1]
Hello, the cross-validation method used in this paper is to use 1-Kold as the training set and the rest as the test set. I want to know that if "For test, train in kfold. Split (train_images, train_labels)" indicates the above operation?  Thank you. Looking forward to your reply.

**Comment by GhTara** [comment]
Hi,
Yes, exactly. I have implemented _10-fold_ in this part that you have mentioned. (in the _function to train model_ section)
Best

**Comment by oula-le** [comment]
Thank you very much for answering my question so quickly. I would like to express my thanks again

**Comment by GhTara** [comment]
You are welcome.
